ArcGISRuntimeSDKQt_CppSamples/Layers/QueryOGCAPICQLFilters/QueryOGCAPICQLFilters.cpp

```diff
@@ -0,0 +1,111 @@
+// [WriteFile Name=Query_OGC, Category=Layers]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "QueryOGCAPICQLFilters.h"
+
+#include "FeatureLayer.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "OgcFeatureCollectionTable.h"
+#include "SimpleLineSymbol.h"
+#include "SimpleRenderer.h"
+
+using namespace Esri::ArcGISRuntime;
+
+QueryOGCAPICQLFilters::QueryOGCAPICQLFilters(QObject* parent /* = nullptr */):
+    QObject(parent),
```

Comment:
Spacing for C++ code in the Qt SDK should be 2 space indentations instead of the 4 (that Qt Creator is set to by default). QML files stay at 4 space indentations